web-registry/src/components/organisms/ChooseCreditClass/ChooseCreditClass.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { useNavigate, useParams } from 'react-router-dom';
+import { Grid } from '@mui/material';
+
+import { ChooseCreditClassGrid } from './ChooseCreditClass.Grid';
+import { ChooseCreditClassItem } from './ChooseCreditClass.Item';
+import { useGetCreditClassOptions } from './hooks/useGetCreditClassOptions';
+import { useErrorTimeout } from './hooks/useErrorTimeout';
+import { validate, getProjectPageBaseData } from '../../../lib/rdf';
+import {
+  useUpdateProjectByIdMutation,
+  useShaclGraphByUriQuery,
+} from '../../../generated/graphql';
+
+const ChooseCreditClass: React.FC = () => {
+  const navigate = useNavigate();
+  const [error, setError] = useErrorTimeout();
+  const { projectId } = useParams();
+  const { creditClassOptions, loading } = useGetCreditClassOptions();
+  const { data: graphData } = useShaclGraphByUriQuery({
+    variables: {
+      uri: 'http://regen.network/ProjectPageShape',
+    },
+  });
+  const [updateProject] = useUpdateProjectByIdMutation();
```

Comment:
not requesting change, but I think it might be good to try and avoid mutations in `organisms`, instead instantiating them in the `/page` and passing down. 

Several reasons here:
- pages are less prone to re-render
- easier to put into storybook
- separates business logic from UI
- helps with UI reuse

I don't think this need to be changed - code looks clean, but we're sort of using a mishmash of atomic design principals and normal code patterns that might complicate things more than either option independently

Actually, now I typed all that I think this component makes more sense as a page, which lives within the `/ChooseCreditClass` folder as a sub-page component alongside the legacy version

Reply:
moved to pages